scripts/benchmark_libsplit.sh

```diff
@@ -6,13 +6,13 @@ fi
 
 BENCHMARK=./build/bin/large
 CURRTIME=`date +"%H-%M-%S_%m-%d-%Y"`
-LOGFILE="./doc/benchmark/logs/$(echo $KDB)_log_$CURRTIME.log"
+#LOGFILE="./doc/benchmark/logs/$(echo $KDB)_log_$CURRTIME.log"
+RUNS=21
 
-
-echo "Runtime as IEEE Std 1003.2-1992 (``POSIX.2'')" >> $LOGFILE
-for run in {0..20}; do
-	echo "RUN: #$run"
-	WALLTIME=$((time -p $BENCHMARK) 2>&1)
-	echo $WALLTIME | awk -F"real" '{print $1;}' # print programs stdout
-	echo $WALLTIME | awk -F"real" '{print $2;}' | awk '{print $1;}' >> $LOGFILE
+echo "Runtime as IEEE Std 1003.2-1992 (``POSIX.2'')"
+echo "Doing $RUNS runs:"
+for run in $(seq 1 $RUNS); do
```

Comment:
seq seems to not exist on OpenBSD #656 

So POSIX would be:

1. count manually: `i=$((i+1));`
2. use awk to generate the seq `awk "BEGIN{for (i = 1; i <= $RUNS; i ++) print i}"`

Maybe you can fix #656, too?